src/js/shield_defs.js

```diff
@@ -3188,35 +3216,44 @@ export function loadShields(shieldImages) {
     },
   };
 
-  shields["US:KY:Parkway"] = Object.assign(
-    {
-      // FIXME: This object contains both spelled-out and abbreviated road
-      // names to accommodate both the abbreviated names from OpenMapTiles and
-      // the spelled-out names from Planetiler.
-      // https://github.com/onthegomap/planetiler/issues/14
-      refsByWayName: {
-        "Audubon Parkway": "AU",
-        "Bluegrass Parkway": "BG",
-        "Bluegrass Pkwy": "BG",
-        "Cumberland Parkway": "LN",
-        "Cumberland Pkwy": "LN",
-        "Hal Rogers Parkway": "HR",
-        "Hal Rogers Pkwy": "HR",
-        "Mountain Parkway": "MP",
-        "Mountain Pkwy": "MP",
-        "Purchase Parkway": "JC",
-        "Purchase Pkwy": "JC",
-        "Western Kentucky Parkway": "WK",
-        "Western Kentucky Pkwy": "WK",
-      },
-    },
-    shields["US:KY:AA"]
-  );
+  shields["US:KY:Parkway"].refsByWayName = {
+    // FIXME: This object contains both spelled-out and abbreviated road
+    // names to accommodate both the abbreviated names from OpenMapTiles and
+    // the spelled-out names from Planetiler.
+    // https://github.com/onthegomap/planetiler/issues/14
+    "Audubon Parkway": "AU",
+    "Bluegrass Parkway": "BG",
+    "Bluegrass Pkwy": "BG",
+    "Cumberland Parkway": "LN",
+    "Cumberland Pkwy": "LN",
+    "Hal Rogers Parkway": "HR",
+    "Hal Rogers Pkwy": "HR",
+    "Mountain Parkway": "MP",
+    "Mountain Pkwy": "MP",
+    "Purchase Parkway": "JC",
+    "Purchase Pkwy": "JC",
+    "Western Kentucky Parkway": "WK",
+    "Western Kentucky Pkwy": "WK",
+  };
 
   shields["US:MI"].overrideByRef = {
     185: diamondShieldBrown,
   };
 
+  shields["US:NY:Parkway"].refsByWayName = {
+    "Bear Mountain Parkway": "BMP",
+    "Bronx and Pelham Parkway": "PP",
+    "Bronx River Parkway": "BRP",
+    "Cross County Parkway": "CCP",
+    "Hutchinson River Parkway": "HRP",
+    "Korean War Veterans Parkway": "KWVP",
+    "Mosholu Parkway": "MP",
+    "Pelham Parkway": "PP",
+    "Saw Mill River Parkway": "SMP",
+    "Sprain Brook Parkway": "SBP",
+    "Taconic State Parkway": "TSP",
+  };
```

Comment:
Just noting here that, aside from the Long Island parkways, these other NYSDOT parkways have been omitted for unavoidable reasons:

* The [Niagara Scenic Parkway](https://www.openstreetmap.org/relation/418069) is currently tagged `network=US:NY`. It has a distinct shield without drop caps, though it would likely simplify down to the same style being used for the other NYSDOT parkways if folks are OK with it saying “NSP” inside anyways.
* The [Lake Ontario State Parkway](https://www.openstreetmap.org/relation/408496) is currently tagged `network=US:NY` `ref=LOSP`. It has a unique shield, quite unlike any of the others.

Let’s ticket out the ones not included here so we don’t lose track of them.

Reply:
Tracked in #516, #517.